k8s/spinnaker/scripts/install/setup_properties.sh

```diff
@@ -22,7 +22,7 @@ if [ -z "$PROJECT_ID" ]; then
 fi
 
 if [ -f "$HOME/click-to-deploy/k8s/spinnaker/scripts/install/properties" ]; then
-  bold "The properties file already exists. Please move it out of the way if you want to generate a new properties file."
+  bold "The properties file already exists at $HOME/click-to-deploy/k8s/spinnaker/scripts/install/properties. Please move it out of the way if you want to generate a new properties file."
```

Comment:
@dorbin: Do you we have any conventions for file paths (e.g. surrounding them with `'` quotes)?